src/WebsocketServer.js

```diff
@@ -309,7 +311,11 @@ module.exports = class WebsocketServer extends events.EventEmitter {
             const connections = stream.getConnections()
 
             connections.forEach((connection) => {
-                connection.send(ControlLayer.BroadcastMessage.create(streamMessage))
+                if (streamMessage.encryptionType !== StreamMessage.ENCRYPTION_TYPES.NONE && connection.messageLayerVersion < 31) {
```

Comment:
For separation of concerns, would be nice if this layer didn't have know anything about the versions. We've managed to do this quite well so far - wouldn't want to break it now.

To avoid hard-coding the info, maybe something simple like 

`StreamMessage.versionSupportsEncryption(connection.messageLayerVersion)` 

could be used, where `versionSupportsEncryption` returns true if `version >= 31`